README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
Yeah so I used to do just `manifest` but I feel like with the new explicit outputs the path to your "binary" ends up being in a different place on concourse than it would locally. So for Java my dist zip (or jar) would live (relative to the root of a project) in `build/distributions/app.zip` when I'm building locally where as if I'm outputting it from a concourse task I'm going to put it in `output-dir/app.zip`. Does that make sense? Am I using the outputs feature wrong? Literally just discovered it yesterday.